inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -686,15 +1010,48 @@ class ScaleFactorPerLayer<InferenceEngine::WeightableLayer*> {
         }
 
         quant->_dst_quant.SetScale(quant->_weights_quant.GetScale() * quant->_src_quant.GetScale());
+        if (quant->_dst_quant.IsStatsSet()) {
+            // Adjust weights scale factor is output values exceed int32 maximum value
```

Comment:
I suppose, here should be "if" instead of "is"

Reply:
you are right.